pkg/agent/util/net_windows.go

```diff
@@ -805,7 +805,7 @@ func NewNetNat(netNatName string, subnetCIDR *net.IPNet) error {
 		}
 	} else {
 		if strings.Contains(internalNet, subnetCIDR.String()) {
-			klog.V(4).InfoS("The existing netnat matched the subnet CIDR", "name", internalNet, "subnetCIDR", subnetCIDR.String())
+			klog.V(4).Infof("The existing netnat matched the subnet CIDR name %s subnetCIDR %s", internalNet, subnetCIDR.String())
```

Comment:
We use structured logging in Antrea.
I am also not sure how this can fix the issue.

Reply:
Same question, did you find the root cause of this issue? @prakrit55